tests/collapse-dtrace.rs

```diff
@@ -0,0 +1,183 @@
+#[macro_use]
+extern crate pretty_assertions;
+
+extern crate inferno;
+
+use inferno::collapse::dtrace::{Folder, Options};
+use inferno::collapse::Collapse;
+use log::Level;
+use std::fs::{self, File};
+use std::io::{self, BufRead, BufReader, Cursor};
+use std::path::Path;
+
+fn test_collapse_perf(test_file: &str, expected_file: &str, options: Options) -> io::Result<()> {
```

Comment:
We should replace `perf` with `dtrace` everywhere in this file.

Reply:
👍